rbd/rbd.go

```diff
@@ -119,7 +120,16 @@ func split(buf []byte) (values []string) {
 
 //
 func (e RBDError) Error() string {
-	return fmt.Sprintf("rbd: ret=%d", e)
+	buf := make([]byte, 1024)
+	// strerror expects errno >= 0
+	errno := math.Abs(float64(e))
```

Comment:
Ick. :-)
Can we avoid this float stuff if possible please? Wouldn't an if-less-than-zero sign flip only be two lines and avoid an import and conversions to float?

Reply:
Sure, I can do that as well.